src/entrypoint/graphql.ts

```diff
@@ -357,7 +357,7 @@ export async function startApolloServer() {
       // i.e. catch-all errors will not be forwarded
       if(log = err.extensions?.exception?.log) {
         const errObj = { message: err.message, code: err.extensions.code }
-        log(errObj)
+        log({...errObj, ...err.extensions.metadata})
```

Comment:
could you expend on what this is doing? (maybe also add some comment in the code to make it a bit more clear?)